lib/portage/dbapi/_MergeProcess.py

```diff
@@ -180,6 +181,15 @@ def _spawn(self, args, fd_pipes, **kwargs):
         self._dblink = mylink
         self._elog_reader_fd = elog_reader_fd
 
+        # Since QueryCommand._db is not entirely picklable, select only
+        # tree types that QueryCommand specifically requires.
```

Comment:
It turns out the the bindbapi._aux_cache_slot_dict instance attribute is not picklable, which needs to be fixed anyway because self.mydbapi can be a bindbapi.

Reply:
Fixed bindbapi, and updated the comment here since QueryCommand._db is now entirely picklable but it's still desirable to filter it.